### DISCUSSION TITLE
potentional bug: the vote power calculation

### DISCUSSION THREAD

**Discussion by longfenghou** [snapshot-labs/snapshot#1098] [General]
hello , i am new to the snapshot. I swaped 100 cake from bnb the 16 Nov, 2021 and then i transfered 50 cake to my friend. Today, i found that i still have 100 cakevote power. ( i know that the cakevote is calculated at the moment of snapshot) However, i think this may not the logic bc i have only 50 cake now. Potential behavior, for some important vote, somebody may collect huge amount of token (from lending platform of other ways) so as to influence the voting result. What do you think of this problem?

**Comment by metrox-eth** [comment]
If someone want to borrow token to vote on a proposal, how to prevent this, this has nothing to do with the moment of the snapshot, on the contrary, it's more difficult to plan a bad behavior if you dont know the content of the proposal and you can't change your voting power once the proposal is created.
